### ISSUE TITLE
Icon for "Path" checkbox is incorrect in projector mode

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/my-solar-system#266]
There's a bug in the icon for the "Path" checkbox.  With the default color profile, it looks like this, using pathIcon.png:

<img width="227" alt="screenshot_2836" src="https://github.com/phetsims/my-solar-system/assets/3046552/54aabc08-76f7-411c-992d-cad599926508">

With the projector color profile, it's supposed to change to use pathIconProjector.png, which looks like this:

<img width="294" alt="screenshot_2838" src="https://github.com/phetsims/my-solar-system/assets/3046552/5581d707-4119-4744-b88d-590fee5ec270">

But in practice, it does not change, and looks like this, which is difficult to see:

<img width="229" alt="screenshot_2837" src="https://github.com/phetsims/my-solar-system/assets/3046552/75f87091-e01e-41c3-a0d7-068ff76fa312">

Here's the relevant code, in MySolarSystemCheckbox.ts:

```ts
  public static createPathCheckbox( pathVisibleProperty: Property<boolean>, tandem: Tandem ): SolarSystemCommonCheckbox {

    const icon = new Image( pathIcon_png, { scale: 0.25 } );
    colorProfileProperty.lazyLink( profileName => {
      assert && assert( profileName === SceneryConstants.DEFAULT_COLOR_PROFILE || profileName === SceneryConstants.PROJECTOR_COLOR_PROFILE );
      icon.setImage( profileName === SceneryConstants.PROJECTOR_COLOR_PROFILE ? pathIconProjector_png : pathIcon_png );
    } );

    return new SolarSystemCommonCheckbox( pathVisibleProperty, SolarSystemCommonStrings.pathStringProperty, {
      icon: new Image( pathIcon_png, { scale: 0.25 } ),
      tandem: tandem
    } );
  }
```

**Comment by pixelzoom** [comment]
This problem is not present in the published version (1.2.3) of My Solar System, so it's a regression.

**Comment by pixelzoom** [comment]
Fixed in the above commit. Closing.
